qdev_wrappers/alazar_controllers/acquisition_parameters.py

```diff
@@ -37,10 +37,9 @@ def set_raw(self, value):
             value: value to set the parameter to
         """
```

Comment:
not sure it's relevant for this PR, but [`NonSettableDerivedParameter`](https://github.com/qdev-dk/qdev-wrappers/blob/3412bc01e13060ea88428a876477700d9e0e5efb/qdev_wrappers/alazar_controllers/acquisition_parameters.py#L70) can just be done via `Parameter('name', instrument=instrument, get_cmd=None, set_cmd=False`, and funnily enough the `alternative` argument and `_alternative` attribute are not used at all, also in that module (but i might be missing stuff)